src/Meta/goddardrocket.jl

```diff
@@ -0,0 +1,25 @@
+goddardrocket_meta = Dict(
```

Comment:
I suggest we rename it `rocket` to match the CUTEst collection of test problems. Can you also rename the files to change `goddardrocket.jl` as `rocket.jl`. Can you change all the `goddardrocket` by `rocket` in this file.